src/main.cpp

```diff
@@ -1,13 +1,21 @@
 #include <Arduino.h>
 #include "mcp2515_can.h"
+#include "can_common.h"
 
 #define SERIAL_SPEED 115200 // one of most common serial port speeds
 #define CAN_CS_PIN  10 // PIN D10
 #define CAN_SPEED CAN_500KBPS // speed of CAN network (fixed for car)
 #define CAN_CONTROLLER_SPEED MCP_8MHz // speed of crystal
-#define DEBUG_MSG_RECEIVE 1 // change to 0 if using battery 
+#define DEBUG_MSG_RECEIVE 1 // change to 0 if using battery
+#define CAN_BATT_SOC_SCALING_FACTOR 2.0
+#define CAN_BATT_VOLTAGE_SCALING_FACTOR 10.0
+
 mcp2515_can can(CAN_CS_PIN); // creating CAN object
 
+// Global varible declaration
+float voltage = 0.0;
+float soc = 0.0;
```

Comment:
As a simple way to tell if CAN receive isn't working, consider making the initial values of these implausible. For example, both voltage and SOC could theoretically be 0, but they should never be negative. So you could initialize them both to -1.0.